src/components/tooltip/GCMarker.js

```diff
@@ -423,80 +430,104 @@ function _treeInsert(
   let node = tree.get(component);
   if (!node) {
     // Make a new node and grow the tree in this direction.
-    node = { branches: new Map() };
+    const value = { name: '(temp)', time: 0 };
```

Comment:
I was thinking of suggesting extracting `(temp)` to a constant variable because it's reused below, or just use `null`, but it's fairly self-contained here so I don't really mind after all. It's up to you!

Reply:
I'm constrained by the type to use a string (I'd rather not relax it to allow `null`; this is an incomplete value here, and final tree nodes should always have a value.)

I think I'll try making a constant temporary node and referring back to it.